### PR TITLE
[#174710415] Send confirmation email to user after data delete

### PR DESCRIPTION
**Pull request by balanza**
Send a confirmation email to a user after his/her data is been deleted. 

### QUESTIONS
* ~Is it correct to send email to every user, regarding the fact that the email si validated or not?~
* ~How to deal the case in which the user has no email?~

### CODE REVIEW

File: GetProfileActivity/handler.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
+/**
+ * Updates the status of a Profile record
```
Comment (gunzip) [inline GetProfileActivity/handler.ts:2]: needs to be updated

File: GetProfileActivity/handler.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    .map(record =>
+      ActivityResultSuccess.encode({
+        kind: "SUCCESS",
+        value: record
+      })
+    )
+    .mapLeft(failure => {
+      logFailure(context)(failure);
+      return failure;
+    })
```
Comment (gunzip) [inline GetProfileActivity/handler.ts:153]: you can use fold and avoid then()

**Review by gunzip — COMMENTED** [review]
I agree that there's a lot of duplicated code that must be avoid (to send emails), it's probably better to resolve this issue then return on this again

**Comment by balanza** [comment]
closed and reopened at https://github.com/pagopa/io-functions-admin/pull/86

**Pull request closed without merging**